server/manifest.go

```diff
@@ -14,7 +14,7 @@ const manifestStr = `
 {
   "id": "com.mattermost.plugin-todo",
   "name": "To Do Plugin",
-  "description": "This plugin makes it easy to keep track of to do items and get daily reminders.",
+  "description": "This plugin makes it easy to keep track of to do issues and get daily reminders.",
```

Comment:
1/5 non-blocking, but I think this should remain `items` in plugin.json.  The code references items exclusively and doesn't mention `issue`

Reply:
The code used to talk about items, but since the big refactoring while introducing "request a task from another user" (https://github.com/mattermost/mattermost-plugin-todo/pull/27 ) they were renamed to issues.